### ISSUE TITLE
Any performance results compared with clBlast?

### ISSUE THREAD

**Issue by liyancas** [ARM-software/ComputeLibrary#449]
**Output of 'strings libarm_compute.so | grep arm_compute_version':**

**Platform:**
ARM Mail GPU or And Adreno GPU

**Operating System:**
Android System

**Problem description:**
NA

**Comment by psyhtest** [comment]
@liyancas Do you mean [clBLAS](https://github.com/clMathLibraries/clBLAS) or [CLBlast](https://github.com/CNugteren/CLBlast)?

**Comment by liyancas** [comment]
CLBlast (https://github.com/CNugteren/CLBlast).

**Comment by psyhtest** [comment]
@liyancas We did it last year when were tuning CLBlast for Firefly-RK3399 and other platforms. Generally, ArmCL was better than CLBlast tuned for the default matrix sizes (e.g. 1024x1024), but we managed to close the gap considerably with advanced tuning techniques and adaptation. Moreover, for some matrix sizes the performance of ArmCL dropped to only half of the peak, while with our modifications CLBlast was able to keep high performance. If it's more than a casual interest, I could provide more details sometime.
